serving/samples/helloworld-java-spark/README.md

```diff
@@ -0,0 +1,181 @@
+# Hello World - Spring Boot Java sample
```

Comment:
need to rename this to `Hello World - Spark Java Example`

Reply:
done